legendary_eureka.py

```diff
@@ -31,9 +33,120 @@ def get_jobs(self):
 
 
 class Glassdoor(Resource):
+
+    JOB_TYPES = ['fulltime', 'parttime', 'contract', 'internship', 'temporary', 'apprenticeship', 'entrylevel']
```

Comment:
Should I change to `__JOB_TYPES` ?
As we don't want this list to be changed..

Reply:
That's totally keep it that way